rkt/api_service.go

```diff
@@ -114,16 +114,9 @@ func isPartOf(keyword, s string) bool {
 	return strings.Contains(s, keyword)
 }
 
-// satisfiesPodFilter returns true if the pod satisfies the filter or the filter
-// is nil, which means the pod will be returned.
-// It returns false if the pod doesn't satisfy the filter, which means it will
-// not be returned.
+// satisfiesPodFilter returns true if the pod satisfies the filter.
+// The pod, manifest, filter must not be nil.
```

Comment:
@jonboulle What do you mean by using types? Btw, the pointers come from the grpc generated code.